pkg/kubeadm/node/csr.go

```diff
@@ -34,7 +35,15 @@ import (
 )
 
 func getNodeName() string {
-	return "TODO"
+	nodename, err := os.Hostname()
```

Comment:
We need this to match whatever kubelet will use to register the node. I'm sure there is logic that determines node name.